### ISSUE TITLE
Document dispatcher methods

### ISSUE THREAD

**Issue by itdependsnetworks** [nautobot/nornir-nautobot#45]
Currently when the dispatched fails, the operator is provided with an error based on ` f"Unable to find the driver for {method} for platform: {task.host.platform}")`. It would be good to add a link into that error message to point to a website of what was supported from a platform and method perspective, similar to the napalm getters.

**Comment by electro47h** [comment]
@itdependsnetworks, are you talking about these methods? This is what I gathered from looking at the class methods.


| | default_nornir | default_netmiko | ASA | FASTIRON | EOS | IOS | IOSXR | JUNOS | NXOS | ROUTEROS | ROUTEROS_API | SMARTZONE_API |
| :-- | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
| check_connectivity | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| compliance_config  | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| generate_config    | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| get_config         | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| provision_config   | ❌ | ✅ | ✅ | ✅ | ❌ | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ | ❌ |

**Comment by itdependsnetworks** [comment]
yes, something like `f"Unable to find the driver for {method} for platform: {task.host.platform}, and expect {netutils.config.compliance.join(', ')}")`
